presto-docs/src/main/sphinx/release/release-0.219.rst

```diff
@@ -0,0 +1,62 @@
+=============
+Release 0.219
+=============
+
+General Changes
+---------------
+
+* Fix a correctness bug for queries with a ``LIMIT`` on top of a ``FULL OUTER JOIN``.
+* Fix a correctness bug in how word boundaries are handled for Joni regular expression functions.
+* Fix Parquet predicate pushdown for ``smallint`` and ``tinyint`` types.
+* Fix a failure when view columns were fully qualified with catalog and/or schema.
+* Fix a failure when creating and reading an unpartitioned Hive table within the same transaction.
+* Fix an issue that would disable a performance optimization if two tables with mismatching but compatible bucket counts are joined whiles filters are applied to one of the tables.
+* Fix an issue that caused procedure calls to fail if no queries were run after the server started up.
+* Fix an issue where analyze properties would remain in the ``system.metadata.analyze_properties`` table even after the connector was removed.
+* Add :func:`ST_Length` for spherical geography
+* Add ``view_owner`` column to the ``information_schema.views`` system table.
+* Add a json version of the query plan to ``QueryCompletedEvent``.
+* Add support for creating warnings during parsing.
+* Add a warning for using reserved word ``current_role`` as an identifier.
+* Add support for using the empty string as a delimiter for the :func:`split` function.
+  When an empty string is used as the delimiter, the string will be split into individual characters.
+
+Raptor Changes
+--------------
+* Add ``raptor.minimum-node-count`` configuration property. The the number of nodes in a cluster is fewer then the configured amount, data recovery and reassignment will not run.
+* Advance the first run of bucket rebalancer. Now bucket balancer will run after a grace period (configured in `raptor.startup-grace-period`) from coordinator restart.
+
+Hive Connector Changes
+----------------------
+
+* Add `hive.ignore-table-bucketing` configuration property and `ignore_table_bucketing` session property.  When set to true these properties enable reading from partitions whose bucketing scheme does not match the table bucketing scheme.
```

Comment:
What does bucketing scheme mean? Is it the columns, the number of buckets or either?

Reply:
@rschlussel

Reply:
either.  we plan the query as though the table is unbucketed.